maven-release-manager/pom.xml

```diff
@@ -284,7 +284,7 @@
           <plugin>
             <groupId>org.codehaus.plexus</groupId>
             <artifactId>plexus-maven-plugin</artifactId>
-            <version>1.3.8</version>
+            <version>${plexus-maven-plugin.version}</version>
```

Comment:
@cstamas Do we need this plugin at all?

Reply:
this plugin has been removed in https://github.com/apache/maven-release/pull/118/files#diff-6b7a6eb88314d4014fb872beb2cc0edf29e6dde4f5f1059be3ff67bf03f3c305L211

Reply:
this plugin has been removed in https://github.com/apache/maven-release/pull/118/files#diff-6b7a6eb88314d4014fb872beb2cc0edf29e6dde4f5f1059be3ff67bf03f3c305L211

Reply:
That commit you mentioned seems to be committed.
Yet I still see this plugin in the master branch
https://github.com/apache/maven-release/blob/master/maven-release-manager/pom.xml#L284

What do you want me to change in this pull request?

Reply:
aaand it's gone